worlds/animal_well/region_data.py

```diff
@@ -608,17 +628,20 @@ class AWData(NamedTuple):
             AWData(AWType.region),
         lname.egg_crystal:
             AWData(AWType.location, [[iname.top, iname.ball, iname.remote, iname.wheel, iname.slink],
-                                     [iname.top, iname.ball, iname.remote, iname.wheel, iname.disc, iname.weird_tricks],  # rooby's version
-                                     [iname.top, iname.wheel_hop, iname.weird_tricks],  # 8's version
-                                     [iname.top, iname.wheel, iname.bubble_long, iname.weird_tricks]]),  # 8's OTHER version
+                                     # rooby's version
+                                     [iname.top, iname.ball, iname.remote, iname.wheel, iname.disc, iname.obscure_tricks],
```

Comment:
Should this be iname.ball_easy?

Reply:
Nope. The full logical implication of this ball is to break the blocks. You can do the rest of this puzzle with the other tools at your disposal.